expression/builtin_op.go

```diff
@@ -403,6 +407,8 @@ func (c *isTrueOrFalseFunctionClass) getFunction(ctx sessionctx.Context, args []
 		case types.ETInt:
 			sig = &builtinIntIsTrueSig{bf}
 			sig.setPbCode(tipb.ScalarFuncSig_IntIsTrue)
+		default:
+			panic("unexpected types.EvalType")
```

Comment:
How about putting the actual `EvalType` into the error message like `fmt.Sprintf("unexpected types.EvalType=%v", argTp)`.